lib/school_house_web/templates/lesson/lesson.html.eex

```diff
@@ -13,10 +13,67 @@
     Caught a mistake or want to contribute to the lesson?
     <a href="https://github.com/elixirschool/elixirschool/edit/master/<%= @lesson.locale %>/lessons/<%= @lesson.section %>/<%= @lesson.name %>.md" target="_blank" rel="noopener">
       Edit this lesson on GitHub!
     </a>
   </blockquote>
 </section>
+
+<section class="bg-brand-light-gray full-width -mb-16 mt-16">
+  <div class="max-w-7xl mx-auto py-8 px-4 sm:px-6 lg:py-12 lg:px-8 items-center">
+      <div class="flex md:justify-between flex-col-reverse md:flex-row">
+        <div class="md:w-1/3 mt-8 md:mt-0">
+          <%= if @lesson.previous do %>
+            <div class="flex md:justify-start justify-center items-center content-center mb-6">
+              <svg xmlns="http://www.w3.org/2000/svg" class="text-gray-400 h-6 w-6 mr-2" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M15 19l-7-7 7-7" />
+              </svg>
+              <h3 class="leading-tight text-sm font-semibold text-gray-400 tracking-wider uppercase">
+                <a href="/<%= @lesson.previous.locale %>/<%= @lesson.previous.section %>/<%= @lesson.previous.name %>" class="hover:text-gray-900" rel="prev">
+                  Previous lesson
+                </a>
+              </h3>
+            </div>
+            <div class="text-center">
+              <p class="text-base font-semibold text-gray-900 mb-2">
+                <%= @lesson.previous.title %>
+              </p>
+              <p class="md:text-left text-center text-sm text-gray-500 overflow-hidden max-h-16">
+                <%= raw @lesson.previous.excerpt %>
+              </p>
+            </div>
+          <% end %>
+        </div>
+        <div class="md:w-1/3">
+          <%= if @lesson.next do %>
+            <div class="flex md:justify-end justify-center items-center content-center mb-6">
+              <h3 class="leading-tight text-sm font-semibold text-gray-400 tracking-wider uppercase">
+                <a href="/<%= @lesson.next.locale %>/<%= @lesson.next.section %>/<%= @lesson.next.name %>" class="hover:text-gray-900" rel="next">
+                  Next lesson
+                </a>
+              </h3>
+              <svg xmlns="http://www.w3.org/2000/svg" class="text-gray-400 h-6 w-6 ml-2" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 5l7 7-7 7" />
+              </svg>
+            </div>
+            <div class="text-center">
+              <p class="text-base font-semibold text-gray-900 mb-2">
+                <%= @lesson.next.title %>
+              </p>
+              <p class="text-italic md:text-left text-center text-sm text-gray-500 overflow-hidden max-h-16">
+                <%= raw @lesson.next.excerpt %>
+              </p>
+            </div>
+          <% end %>
+        </div>
+      </div>
+  </div>
+</section>
```

Comment:
If _you_ wanted we could pull this out into a partial `lib/school_house_web/templates/lesson/_pagination.html.eex`, then update this to:

```eex
<%= render("_pagination.html", previous: @lesson.previous, next: @lesson.next) %>
```

You would need to update the markup so `@lesson.next` became `@next` etc.